lib/TableGen/Operations.cpp

```diff
@@ -228,12 +237,34 @@ void AccessorBuilder::emitSetterDefinition() const {
                 &m_fmt);
 }
 
+void AccessorBuilder::emitVarArgReplacementDefinition(
+    const size_t numNonVarArgs) const {
+  std::string toLlvm = m_arg.name;
+
+  m_fmt.addSubst("numNonVarargs", std::to_string(numNonVarArgs));
+
+  m_os << tgfmt(R"(
+
+      $_op *$_op::replace$0AndInvalidate(::llvm_dialects::Builder &B, ::llvm::ArrayRef<Value *> $1) {
+        ::llvm::SmallVector<Value *> newArgs;
+        if ($numNonVarargs > 0)
+          newArgs.append(arg_begin(), arg_begin() + $numNonVarargs);
+        newArgs.append($1.begin(), $1.end());
+        $_op *newOp = cast<$_op>(B.CreateCall(getCalledFunction(), newArgs, this->getName()));
+        this->replaceAllUsesWith(newOp);
+        this->eraseFromParent();
+        return newOp;
+      })",
+                &m_fmt, convertToCamelFromSnakeCase(toLlvm, true), toLlvm);
```

Comment:
The toLlvm formatting arguments here are not needed, you can use `$name` instead of `$1`